doc/source/user_guide/sampling_wrappers.md

```diff
@@ -1,14 +1,14 @@
 (wrapper_guide)=
 # Sampling wrappers
-Sampling wrappers allow ArviZ to call PPLs in order to perform a limited
+Sampling wrappers allow ArviZ to call PPLs(Probabilistic programming library) in order to perform a limited
 subset of their capabilities and calculate stats and diagnostics that require
 refitting the model on different data.
 
-Their implementation is still experimental and may vary in the future. In fact
+Their implementation is still experimental and may vary in the future. In fact,
 there are currently two possible approaches when creating sampling wrappers.
 The first one delegates all calculations to the PPL
 whereas the second one externalizes the computation of the pointwise log
-likelihood to the user who is expected to write it with xarray/numpy.
+likelihood to the user who is expected to write it with xarray/Numpy.
```

Comment:
All your other PRs enforce the NumPy format which I believe is the correct one, as it' being updated it should be updated to that